### ISSUE TITLE
StyleAvatar Training Resizes Image Wrong

### ISSUE THREAD

**Issue by oijoijcoiejoijce** [LizhenWangT/StyleAvatar#14]
Running full style avatar but it seems to resize images the wrong way. It zooms out a lot on the face for each image (even though it was 1536 crop size from faceverse and there is a lot of space around the face). This makes most of the input images off. 

Is there a mistake or is this normal?

**Comment by LizhenWangT** [comment]
Maybe it's normal. Full styleavatar will crop the image randomly. Can you show me an example image?

**Comment by LizhenWangT** [comment]
Please check if your back image is 1536. The preprocessing code has also been updated.

**Comment by LizhenWangT** [comment]
It's normal. And I forgot to adjust the loss term weights in this version. Please change the line for vgg_loss:
```vgg_loss = vgg(F.upsample(fake_img, size=(512, 512), mode='bilinear'), image_512) * 0.01```
New code will be updated soon.
